swap_meet/vendor.py

```diff
@@ -1,2 +1,57 @@
 class Vendor:
-    pass
\ No newline at end of file
+    
+    def __init__(self,inventory=None):
+        self.inventory = inventory if inventory is not None else []
+
+    def add(self,item):
+        self.inventory.append(item)
+        return item
+
+    def remove(self,item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        return False
```

Comment:
Prefer writing the error handling case as a guard clause first, which lets us then emphasize the main part of the code by not indenting it.

```python
if item not in self.inventory: 
    return False 

self.inventory.remove(item) 
return item